ddl/partition.go

```diff
@@ -1586,11 +1586,6 @@ func onAlterTablePartition(t *meta.Meta, job *model.Job) (ver int64, err error)
 	pstate := ptInfo.GetStateByID(partitionID)
 	switch pstate {
 	case model.StatePublic:
-		err = infosync.PutRuleBundles(nil, []*placement.Bundle{bundle})
-		if err != nil {
-			job.State = model.JobStateCancelled
-			return 0, errors.Wrapf(err, "failed to notify PD the placement rules")
-		}
 		ptInfo.SetStateByID(partitionID, model.StateGlobalTxnOnly)
 		// used by ApplyDiff in updateSchemaVersion
 		job.CtxVars = []interface{}{partitionID}
```

Comment:
doesn't need it any more

Reply:
Why? I think this shouldn't be omitted. We need to pass it to update the state in `infoschema`.

Reply:
It is passed to update placement rule cache, but you did not call `infosync.PutRuleBundles`, right? Then placement rule cache does not need to be updated.

Reply:
> It is passed to update placement rule cache, but you did not call infosync.PutRuleBundles, right?

Not only update the `placement rule cache`. In `ApplyDiff`, we will also update the tableInfo in `infoschema`. In the first phase (`Public -> GlobalTxnOnly`),  though the placement rule cache don't need to be updated, the partition state is still need to be updated.

Reply:
Thanks for explaining. I got it.

Reply:
ok, seems the `applyPlacementUpdate` in apply diff will be applied twice